life_server/Functions/MySQL/fn_bool.sqf

```diff
@@ -1,26 +1,20 @@
 /*
-    File: fn_bool.sqf
-    Author: TAW_Tonic
-
+    File:	fn_bool.sqf
+    Date:   2017-03-22 17:46:27
+    Author: Patrick "Lucian" Schmidt
+    
     Description:
-    Handles bool conversion for MySQL since MySQL doesn't support 'true' or 'false'
-    instead MySQL uses Tinyint for BOOLEAN (0 = false, 1 = true)
+    Converts Bool to Number and Number to bool
 */
-private ["_bool","_mode"];
-_bool = [_this,0,0,[false,0]] call BIS_fnc_param;
-_mode = [_this,1,0,[0]] call BIS_fnc_param;
 
-switch (_mode) do {
-    case 0: {
-        if (_bool isEqualType 0) exitWith {0};
-        if (_bool) then {1} else {0};
-    };
+params[
+    ["_in", "", [0,true]]
+];
+
+if (_in isEqualType true) exitWith {
+    [0,1] select _in;
+};
 
-    case 1: {
-        if (!(_bool isEqualType 0)) exitWith {false};
-        switch (_bool) do {
-            case 0: {false};
-            case 1: {true};
-        };
-    };
+if (_in isEqualType 0) exitWith {
+    [false,true] select _in;
```

Comment:
I just tested my MRES VS oldMRES.
https://pastebin.com/03bRQPFS
You can see the code and you can use it if you want or not.
I am not gone commit more stuff to this since you guys just complain about if-else or exitWith and dont realize how mutch faster this code is already.